### PR TITLE
Scope limitation

### PR DESCRIPTION
**Pull request by ihlar**
Tried to clarify the scope as per issue #22

### CODE REVIEW

**Review by mjoras — APPROVED** [review]

File: documents/charter.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ While mobile network operators continuously invest in network resources, includi
...
+The Secure Communication of Network Properties (SCONE-PRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as the maximum achievable bandwidth for a video. The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video using QUIC connections carrying those applications. Further use cases may be considered later in the working group, however, it is not assumpted that all signaling use cases must or can be addressed by the same protocol. Any protocol specified by the working group should be based on a clearly defined use case. Design of a generic signaling protocol is explicitly out of scope for the working group.
```
Comment (SpencerDawkins) [inline documents/charter.md:7]: I think this is right. Since there's a PR, we should probably correct "assumpted" as "assumed".

**Review by SpencerDawkins — APPROVED** [review]
Thank you!

**Comment by zaheduzzaman** [comment]
you need to clarify or give example of signaling protocol. It is not clear to me what exactly the "signaling protocol" means here, does it mean the signal to use sconepro or does it mean signal to covey say the periodic available bandwidth update?

**Comment by ihlar** [comment]
> you need to clarify or give example of signaling protocol. It is not clear to me what exactly the "signaling protocol" means here, does it mean the signal to use sconepro or does it mean signal to covey say the periodic available bandwidth update?

Sure, will make a stab at that.  
But basically what we mean is that the working group should focus on building a protocol to satisfy the specific use-case of informing clients about video bitrate policies in the network.  The working group should not explicitly try to make a generic protocol that can cater a multitude of other signaling use-cases (such as expressing packet priorities or whatever). Basically, we make no guarantees that the protocol specified is useful for any potential future use case.

**Review by mjoras — APPROVED** [review]

**Review by SpencerDawkins — APPROVED** [review]
I agree that removing the mention of "signaling" addresses @zaheduzzaman's immediate comment, but we should still clarify whether one protocol covers both session establishment and delivery of adaptation guidance to the client. I created #33 to address that.

**Pull request merged**